cmd/maya-apiserver/spc-watcher/start.go

```diff
@@ -38,12 +38,14 @@ var (
 	kubeconfig string
 )
 
+// QueueLoad struct holds object to be pushed in workqueue
 type QueueLoad struct {
 	Key       string
 	Operation string
 	Object    interface{}
 }
 
+// Start function will setup OpenEBS and Kubernetes clientset and launches the SPC controller.
```

Comment:
`Start sets up OpenEBS and Kubernetes clientset and launches the SPC controller.`